web/app/controller/Root.js

```diff
@@ -81,14 +81,25 @@ Ext.define('Traccar.controller.Root', {
                 this.asyncUpdate(true);
             }
         });
-        Ext.get('attribution').remove();
+        if (Ext.get('attribution') !== null) Ext.get('attribution').remove();
```

Comment:
I just noticed this line. First of all, call it once to avoid multiple searches. Also, you need to add curly braces here.